src/context/WalletProvider/KeepKey/components/Label.tsx

```diff
@@ -32,7 +32,7 @@ export const KeepKeyLabel = () => {
       moduleLogger.error(e)
       toast({
         title: translate('common.error'),
-        description: e?.message ?? translate('common.somethingWentWrong'),
+        description: e?.message.message ?? translate('common.somethingWentWrong'),
```

Comment:
```suggestion
        description: e?.message?.message ?? translate('common.somethingWentWrong'),
```

I guess `e.message` can also be undefined in some situations isn't it?

Reply:
true